src/main/scala/org/embulk/output/s3_parquet/S3ParquetOutputPlugin.scala

```diff
@@ -86,8 +86,20 @@ object S3ParquetOutputPlugin
         @Config("catalog")
         @ConfigDefault("null")
         def getCatalog: Optional[CatalogRegistrator.Task]
+
+        @Config("type_options")
+        @ConfigDefault("{}")
+        def getTypeOptions: JMap[String, TypeOptionTask]
     }
 
+    trait ColumnOptionTask extends Task with TimestampColumnOption with LogicalTypeOption
```

Comment:
This trait is used when parsing/dumping the nested option, so you don't need to extend `Task`.

Reply:
I had forgotten to explain ... I found an `java.lang.IllegalArgumentException` issue when I gave nested config value and extending `Task` will solve it.

It's a reproducable code:
https://github.com/syucream/embulk-output-s3_parquet/tree/config-nested
with using docker, like this:
```
$ docker build -t syucream/embulk-output-s3_parquet-config-nested .
...
$ docker run -it syucream/embulk-output-s3_parquet-config-nested
...
Error: org.embulk.config.ConfigException: java.lang.IllegalArgumentException: Can not construct instance of org.embulk.spi.time.TimestampFormatter$TimestampColumnOption, problem: abstract types either need to be mapped to concrete types, have custom deserializer, or be instantiated with additional type information
 at [Source: N/A; line: -1, column: -1] (through reference chain: java.util.LinkedHashMap["t"])
```

Reply:
Do you know what it is? (I guess I had any mistake or there's any other way to process it correctly ...

Reply:
I will respond to this after the investigation.

Reply:
Oh... I'm sooooooooooooooooooooooooo sorry........ I seem to have misunderstood the specification of `TimestampColumnOption`. It was necessary to extend `Task` interface even if it was a nested option... This is my mistake. I'm so sorry.

Reply:
No problem, thanks for your investigation! It's a complicated part in embulk and I wonder there's any better way or not ... anyway I keep it. 😃